src/main/java/tech/thatgravyboat/repolib/api/PetsAPI.java

```diff
@@ -111,7 +111,8 @@ public List<String> getFormattedLore(int level, @Nullable String heldItem) {
                 return this.lore.stream()
                         .map(line -> VARIABLE_PATTERN.matcher(line).replaceAll(match -> {
                             var key = match.group(1);
-                            return String.format("%.1f", this.getStat(key, level, heldItem));
+                            double stat = this.getStat(key, level, heldItem);
+                            return (stat == Math.floor(stat)) ? String.format("%.0f", stat) : String.format("%.1f", stat);
```

Comment:
think something like this should also work, tho the format shouldn't be created here, but github doesn't allow me to go outside of changed things.

```suggestion
                            DecimalFormat format = new DecimalFormat("0.##");
                            double stat = this.getStat(key, level, heldItem);
                            return format.format(stat);
```